config/Makefile.depend

```diff
@@ -332,9 +332,10 @@ MLPLIBEXTRA             = lib/libHist.lib lib/libMatrix.lib lib/libTree.lib \
                           lib/libTreePlayer.lib lib/libMathCore.lib
 SPECTRUMLIBEXTRA        = lib/libHist.lib lib/libMatrix.lib
 UNFOLDLIBEXTRA          = lib/libHist.lib lib/libMatrix.lib lib/libXMLParser.lib
-TMVALIBEXTRA            = lib/libRIO.lib lib/libHist.lib lib/libMatrix.lib \
-                          lib/libTree.lib lib/libGraf.lib lib/libGpad.lib \
-                          lib/libTreePlayer.lib  lib/libMLP.lib \
+TMVALIBEXTRA            = lib/libCore.lib lib/libNet.lib lib/libMultiProc.lib \
```

Comment:
libCore is always implicit, no need to mention it.

Reply:
Ok, good to know.